src/features/points/GiveReceived.tsx

```diff
@@ -111,16 +120,66 @@ export const GiveReceived = ({
     );
   }
 
-  if (size === 'large') {
+  if (size === 'large' || size === 'xl') {
     return (
       <>
-        {size == 'large' && (
-          <>
+        <Panel
+          css={{
+            flexDirection: 'row',
+            p: 0,
+            border: 'none',
+            overflow: 'clip',
+            background: size == 'xl' ? '$surface' : 'transparent',
+          }}
+        >
+          {size == 'xl' && (
+            <Button
+              className="clickProtect"
+              as="span"
+              noPadding
+              color="cta"
+              css={{
+                pointerEvents: 'none',
+                cursor: 'default',
+                p: '$md',
+                borderRadius: 0,
+                background: 'linear-gradient(.33turn, $cta 23%, $complete)',
+              }}
+            >
+              <Flex column css={{ alignItems: 'center', gap: '$sm' }}>
+                <Flex row css={{ alignItems: 'center', gap: '$xs' }}>
+                  <GemCoFill fa size="xl" />
+                  <Text semibold size="large" css={{ textAlign: 'center' }}>
+                    {data.length}
+                  </Text>
+                </Flex>
+                {/* <Text semibold size="xs" css={{ textAlign: 'center' }}>
+                  GIVE <br />
+                  Received
+                </Text> */}
+              </Flex>
+            </Button>
+          )}
+          <Flex
+            css={{
+              columnGap: '$sm',
+              rowGap: 0,
+              flexWrap: 'wrap',
+              m: size == 'xl' ? '$sm' : 0,
+              alignItems: 'flex-start',
+            }}
+          >
             {sortedGives.map(g => (
-              <Flex key={`give_${g.skill}`} css={{ gap: '$md' }}>
+              <Flex key={`give_${g.skill}`}>
```

Comment:
https://legacy.reactjs.org/docs/lists-and-keys.html#keys

> The best way to pick a key is to use a string that uniquely identifies a list item among its siblings. Most often you would use IDs from your data as keys:

I don't think this flex needs a key since it has no siblings

Reply:
I was seeing bugs when it didn't have a key because of the map